NeoML/Python/neoml/Dnn/DnnDistributed.py

```diff
@@ -91,38 +96,46 @@ def last_losses(self, layer_name):
         """Gets values of the loss function on the last step for all models.
 
         :param layer_name: The name of the loss layer for which last losses will
-            be returned. The class of the layer with that name must be `Loss`.
+            be returned. The class of the layer with that name must be `neoml.Dnn.Loss`.
         :type layer_name: str
+
+        .. rubric:: Layer outputs:
+
+            The array of losses for all models.
         """
-        return self._last_losses(layer_name)
+        return self._last_losses(str(layer_name))
 
     def get_output(self, layer_name):
         """Returns last blobs of `layer_name` for all models.
 
         :param layer_name: The name of the layer for which last output will be returned.
             `layer_name` should correspond to neoml.Sink.
         :type layer_name: str
+
+        .. rubric:: Layer outputs:
```

Comment:
It isn't a layer.
The Sphinx has the `:rtype:` annotation for that cases.
+ In one case you return generator (or list?) in another - numpy array. That should be clarified too.
P.S. (same goes for get_last_losses)